android-slack/src/test/java/com/genymobile/android_slack/ExampleUnitTest.java

```diff
@@ -0,0 +1,185 @@
+package com.genymobile.android_slack;
+
+import com.genymobile.android_slack.impl.ChannelsListSlackResponse;
+import com.genymobile.android_slack.impl.RtmStartSlackResponse;
+import com.genymobile.android_slack.impl.SlackResponse;
+import com.genymobile.android_slack.impl.SlackService;
+
+import okhttp3.OkHttpClient;
+import okhttp3.Request;
+import okhttp3.ResponseBody;
+import okhttp3.ws.WebSocket;
+import okhttp3.ws.WebSocketCall;
+import okhttp3.ws.WebSocketListener;
+import okio.Buffer;
+import org.junit.Test;
+import retrofit2.Call;
+import retrofit2.GsonConverterFactory;
+import retrofit2.Response;
+import retrofit2.Retrofit;
+
+import java.io.IOException;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.junit.Assert.assertEquals;
+
+/**
+ * To work on unit tests, switch the Test Artifact in the Build Variants view.
+ */
+public class ExampleUnitTest {
+    
+    // Token issued when creating a bot user in slack interface
+    public static final String TOKEN = "xoxb-18190368645-ij2XqxuaOnXkeDtXKuG9tYpB";
+    public static final String SLACK_API_BASE_URL = "https://slack.com/api/";
+
+    @Test
+    public void addition_isCorrect() throws Exception {
+        assertEquals(4, 2 + 2);
+    }
+
+    @Test
+    public void testSimpleConnection() throws Exception {
```

Comment:
Do not really care :) This file would be delete soon. These are not real tests, it is just some running code to test and evaluate the tools (retrofit, okhttp-ws)